programs/bmptk/Makefile.inc

```diff
@@ -427,7 +427,7 @@ define Windows
    PREFIX            := $(GCC-WIN)/bin/
    TARGET_CPP_FLAGS  += -mno-ms-bitfields   
    TARGET_C_FLAGS    += -mno-ms-bitfields   
-   TARGET_LN_FLAGS   += -lgdi32
+   #TARGET_LN_FLAGS   += -lgdi32
```

Comment:
Is there a reason to disable the windows graphics device interface [(GDI)](https://docs.microsoft.com/en-us/windows/desktop/gdi/windows-gdi) on windows machines?

Reply:
Yes due to some strange thing in HWLIB and bmptk it is not necessary and it still adds the flag running native on linux which gives problems
it is a temporary fix for now